after-an-exploit.md

```diff
@@ -1,25 +1,25 @@
 
 ### Sidebar: Documentation and learning
-One of the first things you should do is open two scratch pads. One for documenting everything you've discovered and that you're doing NOW. If you notice an important detail proving how the attackers got in, add the source of the information and as much of that information as you can. The second document is for things you want to do to harden your site that you think of as you go through the process. You may realize a lot of things you can do to make it harder for attackers, but their not easy or critical enough to do right now. Save those valuable ideas for later.
+One of the first things you should do is open two scratch pads. One for documenting everything you've discovered and that you're doing NOW. If you notice an important detail proving how the attackers got in, add the source of the information as much as you can. The second document is for things you want to do to harden your site that you think of as you go through the process. You may realize a lot of things you can do to make it harder for attackers, but they're not easy or critical enough to implement right now. Save those valuable ideas for later.
 
 ## Step 3: Who should you notify?
-You should start deciding who you need to notify about the issue. If your site had users and you believe your site was completely taken over then private content like their email address, IP address and anything else private on the site is compromised. In some locations you are legally bound to inform people of this exposure (e.g. in the case of sites requiring HIPAA or PCI Compliance), if not also morally bound.
+You should start deciding who you need to notify about the issue. If your site had users and you believe your site was completely taken over, any private content like their email address, IP address and other private information on the site is compromised. In some locations you are legally bound to inform people of this exposure (e.g. in the case of sites requiring HIPAA or PCI Compliance). Even if you're not legally required to do so, please also notify them in a morally bound.
```

Comment:
I think this is an improvement, but the last sentence is a bit mangled at this point. Could you try rewriting it? If not just let me know and I'm happy to come up with ideas.